packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -220,16 +242,40 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
         </g>,
       );
     }
-    if (total === 0) {
+    if (barValue === 0) {
```

Comment:
what does this represent. Can you add comments in the code.

Reply:
renamed the variable for easy interpretation